src/core/mod.rs

```diff
@@ -61,7 +61,7 @@ pub const CHAINSTATE_VERSION: &'static str = "23.0.0.0";
 pub const MICROSTACKS_PER_STACKS: u32 = 1_000_000;
 
 pub const POX_PREPARE_WINDOW_LENGTH: u32 = 240;
```

Comment:
@kantai shouldn't this match `PREPARE_CYCLE_LENGTH` in `pox-mainnet.clar` above? Happy to send a separate PR if so.